app/views/home/how_government_works.html.erb

```diff
@@ -1,534 +1,489 @@
 <% page_title "How government works" %>
 <% page_class "inside_gov_how-gov-works" %>
 
-<div id="how-government-works">
-  <header class="govuk-width-container">
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-three-quarters">
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <header>
         <%= render "govuk_publishing_components/components/title", {
           title: "How government works",
         } %>
         <p class="govuk-body-l">In the UK, the Prime Minister leads the government with the support of the Cabinet and ministers. You can find out <%= link_to("who runs government", "#who-runs-government", class: "govuk-link") %> and <%= link_to("how government is run", "#how-government-is-run", class: "govuk-link") %>, as well as learning about the <%= link_to("history of government", "#history-uk-government", class: "govuk-link") %>.</p>
```

Comment:
This could be swapped for a [leading paragraph Component](https://components.publishing.service.gov.uk/component-guide/lead_paragraph)

Reply:
Looks like you can use a `sanitize` function with escaped quotes to include html in the text property.